code/modules/mob/living/simple_animal/hostile/russian.dm

```diff
@@ -1,11 +1,11 @@
-/mob/living/simple_animal/hostile/russian
-	name = "Russian"
-	desc = "For the Motherland!"
+/mob/living/simple_animal/hostile/soviet
+	name = "Soviet"
+	desc = "For the Union!"
```

Comment:
You have to update this PR with the latest changes that master had, no need to do manual stuff like this.